lib/dispatcher.lib.php

```diff
@@ -304,6 +304,23 @@ public function upkeep($dispatcher, &$reqData, &$out) {
 		}
 	}
 
+	public function loginsearch($dispatcher, &$reqData, &$out) {
+		global $psdb;
+		$server = $dispatcher->findServer();
+		if (!$server || $server['id'] !== 'showdown') {
+			$out['actionerror'] = 'access denied';
+		} else if (!isset($_REQUEST['search'])) {
+			$out['actionerror'] = 'Specify a valid search.';
+		} else {
+			$search = $psdb->escape($_REQUEST['search']);
+			$result = $psdb->query(
+				"SELECT `userid`, `ip` FROM `ntbb_users` WHERE `ip` LIKE '" . $search . "%' " .
+				"OR `userid` LIKE '%" . $search . "%';"
```

Comment:
We do not have fulltext indexes... the database can't handle this kind of query. :[